mywebsite/news/models.py

```diff
@@ -27,11 +28,16 @@ class Status(models.TextChoices):
     status = models.CharField(
         max_length=2, choices=Status.choices, default=Status.DRAFT
     )
+
+    objects = models.Manager()
+    published = PublishedManager()
     class Meta:
         ordering = ["-publish"]
         indexes = [
             models.Index(fields=["-publish"]),
         ]
 
+    def get_absolute_url(self):
+        return reverse("news:article_detail", args=[self.id])
     def __str__(self):
```

Comment:
Между методами должна быть 1 пустая строка.
```suggestion
        return reverse("news:article_detail", args=[self.id])
        
    def __str__(self):
```